.gitignore

```diff
@@ -1 +1,2 @@
 /out/
+*.iml
```

Comment:
No objection to putting it here as well, but I use a global ignore file for my IDE stuff. You can check the location with
```shell
git config --get core.excludesfile
```